files/base.rb

```diff
@@ -45,6 +45,19 @@ def runbook
     @event['check']['runbook'] || false
   end
 
+  def event_description
+    @event['check']['description'] ? @event['check']['description'] : 'N/A'
+  end
+
+  def component
+    component = @event['check']['component']
+    if component
+      !component.empty? ? '[' + component.join(',') + ']' : 'N/A'
```

Comment:
is component guaranteed to be list-like type? tbh i wouldn't bother with N/A and just replaced the whole method with

    def component
      "[" << Array(@event['check']['component'] || []).join(',') << "]"
    end